pyaedt/icepak.py

```diff
@@ -2197,18 +2240,18 @@ def create_pcb_from_3dlayout(
         return status
 
     @pyaedt_function_handler()
-    def copyGroupFrom(self, groupName, sourceDesign, sourceProject=None, sourceProjectPath=None):
+    def copyGroupFrom(self, group_name, source_design, source_project_name=None, source_project_path=None, **kwargs):
         """Copy a group from another design.
 
         Parameters
         ----------
-        groupName : str
+        group_name : str
             Name of the group.
-        sourceDesign : str
+        source_design : str
             Name of the source design.
-        sourceProject : str, optional
+        source_project_name : str, optional
```

Comment:
For this parameter description and the next, it would be helpful to describe what the behavior is when ``None`` is used.
"The default is ``None``, in which case ...."

Reply:
Done.